### PR TITLE
Updated pt-br translation

### PR DESCRIPTION
**Pull request by LucasVieiraS**
Added new pt-br words that were missing from the translation list

### CODE REVIEW

File: packages/tldraw/src/translations/pt-br.json [hunk 1]
```diff
@@ -83,5 +97,28 @@
...
+  "language": "Idioma",
+  "image": "Imagem",
+  "keyboard.shortcuts": "Atalhos de Teclado",
+  "dock.position": "Posição de Ferramentas",
+  "bottom": "Inferior",
+  "top": "Superior",
+  "right": "Direita",
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:109]: Sounds weird when done a literal translation

Suggestion:
```suggestion
  "align.center.x": "Alinhar ao centro na horizontal",
```

File: packages/tldraw/src/translations/pt-br.json [hunk 2]
```diff
@@ -83,5 +97,28 @@
...
+  "distribute.x": "Distruibuir Horizontal",
+  "distribute.y": "Distruibuir Vertical",
+  "stretch.x": "Esticar Horizontal",
+  "stretch.y": "Esticar Vertical",
+  "dialog.save.firsttime": "Você deseja salvar seu projeto atual?",
+  "dialog.save.again": "Você deseja salvar as mudanças ao projeto atual?",
+  "dialog.cancel": "Cancelar",
+  "dialog.no": "Não",
+  "dialog.yes": "Sim",
+  "enter.file.name": "Insira o Nome do Arquivo"
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:123]:
```suggestion
  "enter.file.name": "Insira o nome do arquivo"
```

File: packages/tldraw/src/translations/pt-br.json [hunk 3]
```diff
@@ -83,5 +97,28 @@
...
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
+  "align.right": "Alinhar Direita",
+  "align.top": "Alinhar Superior",
+  "align.center.y": "Alinhar Centro Vertical",
+  "align.bottom": "Alinhar Inferior",
+  "distribute.x": "Distruibuir Horizontal",
+  "distribute.y": "Distruibuir Vertical",
+  "stretch.x": "Esticar Horizontal",
+  "stretch.y": "Esticar Vertical",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:117]:
```suggestion
  "stretch.y": "Esticar na vertical",
```

File: packages/tldraw/src/translations/pt-br.json [hunk 4]
```diff
@@ -83,5 +97,28 @@
...
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
+  "align.right": "Alinhar Direita",
+  "align.top": "Alinhar Superior",
+  "align.center.y": "Alinhar Centro Vertical",
+  "align.bottom": "Alinhar Inferior",
+  "distribute.x": "Distruibuir Horizontal",
+  "distribute.y": "Distruibuir Vertical",
+  "stretch.x": "Esticar Horizontal",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:116]:
```suggestion
  "stretch.x": "Esticar na horizontal",
```

File: packages/tldraw/src/translations/pt-br.json [hunk 5]
```diff
@@ -83,5 +97,28 @@
...
+  "right": "Direita",
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
+  "align.right": "Alinhar Direita",
+  "align.top": "Alinhar Superior",
+  "align.center.y": "Alinhar Centro Vertical",
+  "align.bottom": "Alinhar Inferior",
+  "distribute.x": "Distruibuir Horizontal",
+  "distribute.y": "Distruibuir Vertical",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:115]:
```suggestion
  "distribute.y": "Distruibuir na vertical",
```

File: packages/tldraw/src/translations/pt-br.json [hunk 6]
```diff
@@ -83,5 +97,28 @@
...
+  "top": "Superior",
+  "right": "Direita",
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
+  "align.right": "Alinhar Direita",
+  "align.top": "Alinhar Superior",
+  "align.center.y": "Alinhar Centro Vertical",
+  "align.bottom": "Alinhar Inferior",
+  "distribute.x": "Distruibuir Horizontal",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:114]:
```suggestion
  "distribute.x": "Distruibuir na horizontal",
```

File: packages/tldraw/src/translations/pt-br.json [hunk 7]
```diff
@@ -83,5 +97,28 @@
...
+  "bottom": "Inferior",
+  "top": "Superior",
+  "right": "Direita",
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
+  "align.right": "Alinhar Direita",
+  "align.top": "Alinhar Superior",
+  "align.center.y": "Alinhar Centro Vertical",
+  "align.bottom": "Alinhar Inferior",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:113]:
```suggestion
  "align.bottom": "Alinhas embaixo",
```

File: packages/tldraw/src/translations/pt-br.json [hunk 8]
```diff
@@ -83,5 +97,28 @@
...
+  "keyboard.shortcuts": "Atalhos de Teclado",
+  "dock.position": "Posição de Ferramentas",
+  "bottom": "Inferior",
+  "top": "Superior",
+  "right": "Direita",
+  "left": "Esquerda",
+  "align.left": "Alinhar Esquerda",
+  "align.center.x": "Alinhar Centro Horizontal",
+  "align.right": "Alinhar Direita",
+  "align.top": "Alinhar Superior",
```
Comment (bybruno) [inline packages/tldraw/src/translations/pt-br.json:111]:
```suggestion
  "align.top": "Alinhas em cima",
```

**Review by bybruno — COMMENTED** [review]
Adding words is necessary to adapt them for better understanding by the end user.

Example: `Alinhar Centro Horizontal` 

it's necessary to add preposition and article: `Alinhar ao centro na horizontal`

**Comment by LucasVieiraS** [comment]
Definitely! Thank you for the suggestions, hadn't taken that in consideration...

I'll do the neccessary changes asap.

**Comment by LucasVieiraS** [comment]
> Adding words is necessary to adapt them for better understanding by the end user.
> 
> Example: `Alinhar Centro Horizontal`
> 
> it's necessary to add preposition and article: `Alinhar ao centro na horizontal`

Definitely! Thank you for the suggestions, hadn't taken that in consideration...

I'll do the neccessary changes asap.

**Comment by steveruizok** [comment]
Thanks @LucasVieiraS and @bybruno!

**Pull request merged**